### ISSUE TITLE
Cleanup and prepare v1.4.0

### ISSUE THREAD

**Issue by mikkonie** [bihealth/sodar-core#1862]
## Minor Tasks

- [x] Rename `No global background jobs found` in bgjobs (1)
- [x] Try tweaking martor editor font weight and preview table cell padding
- [x] Reorder `Makefile` targets
- [x] Add note on `django-markupfield` in `base.txt` (4)
- [x] Fix `ProjectInviteRevokeView` docstring
- [x] `adminalerts/tests/test_plugins.py`: update docstrings
- [x] `adminalerts/tests/test_views_ajax.py`: Fix `test_get()` docstring
- [x] `filesfolders/plugins.py`: Fix line breaks from redundant comma separation in single-arg lists
- [x] `filesfolders/tests/test_models.py`: Standardize test func names and docstrings
- [x] `filesfolders/tests/test_ui.py`: Standardize test func names and docstrings
- [x] `filesfolders/tests/test_permissions_api.py`: Remove redundant `'s` in test data descriptions
- [x] `filesfolders/tests/test_views_api.py`: Remove redundant `'s` in test data descriptions
- [x] `projectroles/management/commands/removeroles.py`: Clarify help texts (3)
- [x] `projectroles/plugins.py`: Revise `search()` docstring, revise RST syntax in docstrings
- [x] `projectroles/templates/projectroles/_messages.html`: Fix long lines
- [x] `projectroles/templates/projectroles/search_results.html`: Fix "global" mention
- [x] `projectroles/templatetags/projectroles_common_tags.py`: Revise markdown warning
- [x] `projectroles/tests/test_app_settings.py`: Capitalize scopes in docstrings
- [x] `projectroles/tests/test_commands.py`: Standardize docstrings
- [x] `projectroles/tests/test_models.py`: Standardize test func names and docstrings
- [x] `projectroles/tests/test_remote_project_api.py`: Standardize docstrings
- [x] `projectroles/tests/test_ui.py`: Standardize docstrings
- [x] `projectroles/tests/test_views.py`: Standardize docstrings
- [x] `projectroles/tests/test_views_ajax.py`: Standardize test func names and docstrings
- [x] `siteinfo/tests/test_permissions.py`: Standardize docstrings
- [x] `siteinfo/tests/test_ui.py`: Standardize docstrings
- [x] `siteinfo/tests/test_views.py`: Standardize docstrings
- [x] `timeline/plugins.py`: Standardize docstrings
- [x] `timeline/tests/test_models.py`: Standardize test func names and docstrings
- [x] `timeline/tests/test_templatetags.py`: Standardize docstrings
- [x] Rename `dropdown_user_name_display` `AppSetting` to `user_dropdown_name_display`

## Documentation

- [x] Specify checking out `dev` branch in `dev_core_install.rst`
- [x] Add Federico in `AUTHORS.rst`
- [x] Fix extra blank lines in changelog
- [x] `docs/source/app_projectroles_settings.rst`: Revise Markdown section
- [x] `docs/source/app_projectroles_settings.rst`: Revise LDAP section intro
- [x] `docs/source/app_projectroles_usage.rst`: Revise "Transfer Roles Between Users" section
- [x] `docs/source/app_projectroles_usage.rst`: Revise search section
- [x] `docs/source/app_userprofile.rst`: Revise username display setting description (2)
- [x] `docs/source/dev_project_app.rst`: Revise "Search API" section
- [x] `docs/source/dev_project_app.rst`: Revise "Search Results" section
- [x] Rename `Integration in an Existing Site` section and link
- [x] `docs/source/index.rst`: Revise
- [x] `docs/source/major_changes.rst`: Reorder and revise Breaking Changes section
- [x] Fix Sphinx v9 warnings (5)

## Issues to Add in CHANGELOG

- [x] Add #22
- [x] Add #1191
- [x] Add #1434
- [x] Add #1513 
- [x] Add #1853
- [x] Add #1550
- [x] Add #1848
- [x] Add #1861
- [x] Add #1869 
- [x] Add #1873
- [x] Add #1882
- [x] Add #1894
- [x] Add #1898
- [x] Add #1899
- [x] Add #1901 
- [x] Add #1902
- [x] Add #1913
- [x] Add #1916
- [x] Add #1917 
- [x] Add #1918
- [x] Add #1923
- [x] Add #1926
- [x] Add #1927
- [x] Optional `request` arg for `modify_assignment()` (done as part of #1875)
- [x] General docs updates (#1884)
- [x] Mention formatting for #1881

## Release Checklist

- [x] Review code style and cleanup if needed
- [x] Review and update doc entries if needed
- [x] Ensure all relevant updates are in `CHANGELOG` and major changes doc
- [x] Ensure REST API versions are up to date and documented
- [x] Ensure versions in `pyproject.toml` are up to date
- [x] Upgrade version number of pypi package reference in `README`
- [x] Upgrade docs config version number (usually at `x.y.z-WIP` when developing)
- [x] Update latest version info in `codemeta.json`
- [x] Update version number and date in `CHANGELOG`
- [x] Update version number and date in `Major Changes` doc
- [x] Ensure docs can be built without errors
- [x] Ensure `make spectacular` runs without errors or warnings (until in CI)

## Notes

1. "Global" has no meaning in this context, these are site-wide jobs.
2. "Owner" is somewhat misleading here, might be confused with the project owner role.
3. Should clarify that we specifically erase the name and email of the user.
4. Not used, needed for migrations, to be removed in v2.1 following migration squash.
5. Getting the following: `<unknown>:1: WARNING: more than one target found for cross-reference 'type': projectroles.models.AppSetting.type, projectroles.models.Project.type [ref.python]`

**Comment by mikkonie** [comment]
Done.
